### PR TITLE
CMR-7259: Added back stac docs link to CMR Search Landing page.

### PR DESCRIPTION
**Pull request by sxu123**

### CODE REVIEW

File: search-app/src/cmr/search/site/data.clj [hunk 1]
```diff
@@ -175,6 +175,14 @@
...
 ;;; Page data functions
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+(defn- get-stac-docs-url
+  "Returns stac docs url based on the stac-base-url."
+  [stac-base-url]
+  (cond
+    (string/includes? stac-base-url "3000") (str stac-base-url "stac/docs")
+    (string/includes? stac-base-url "sit") "https://d3se4zjsrvasc2.cloudfront.net/docs/index.html"
+    (string/includes? stac-base-url "uat") "https://d346wn5qqiom8e.cloudfront.net/docs/index.html"
+    :else "https://d3hqnx562z478b.cloudfront.net/docs/index.html"))
```
Comment (jaybarra) [inline search-app/src/cmr/search/site/data.clj:185]: Do we have any way to convert this to a nicer looking path?

**Review by jaybarra — APPROVED** [review]
Having it be available is a start, but it appears to be missing functionality in the side panel.

File: search-app/resources/templates/stac-search-docs.html [hunk 2]
```diff
@@ -3,10 +3,11 @@
 {% block main-content %}
 <h2>CMR STAC Links</h2>
 <p class="lead">
-   CMR STAC Links contain two parts:
+   CMR STAC Links contain three parts:
```
Comment (ygliuvt) [inline search-app/resources/templates/stac-search-docs.html:6]: nit: I don't think this sentence is necessary.

File: search-app/resources/templates/stac-search-docs.html [hunk 3]
```diff
@@ -3,10 +3,11 @@
...
 </p>
 <ul>
   <li><a href="{{ stac-url }}">CMR STAC</a> - CMR STAC Catalog Endpoint</li>
   <li><a href="{{ cloudstac-url }}">CMR Cloud STAC</a> - CMR STAC Catalog Endpoint for Cloud-hosted Holdings Only</li>
+  <li><a href="{{ stac-docs-url }}">CMR STAC Documentation</a> - CMR STAC Catalog Documentation</li>
```
Comment (ygliuvt) [inline search-app/resources/templates/stac-search-docs.html:11]: nit: I would put the documentation link as the first link.

File: search-app/src/cmr/search/site/data.clj [hunk 4]
```diff
@@ -175,6 +175,14 @@
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 ;;; Page data functions
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+(defn- get-stac-docs-url
+  "Returns stac docs url based on the stac-base-url."
+  [stac-base-url]
+  (cond
+    (string/includes? stac-base-url "3000") (str stac-base-url "stac/docs")
+    (string/includes? stac-base-url "sit") "https://d3se4zjsrvasc2.cloudfront.net/docs/index.html"
```
Comment (ygliuvt) [inline search-app/src/cmr/search/site/data.clj:183]: I don't think this is allowed.

**Review by ygliuvt — CHANGES_REQUESTED** [review]
Make internal urls config parameters.

**Review by ygliuvt — APPROVED** [review]

**Pull request merged**